components/FormulaireUnicorn.vue

```diff
@@ -0,0 +1,116 @@
+<template>
+    <div class="formulaire-unicorn">
+        <label for="name" class=""> Name </label><br/>
+        <input type="text" name="name" id="name" autocomplete="off" :placeholder="unicorn.name" class="formulaire-unicorn__input"> <br/>
+        <label for="birthyear"> Birthyear </label> <br/>
+        <input type="text" name="birthyear" id="birthyear" autocomplete="off" :placeholder="unicorn.birthyear" class="formulaire-unicorn__input"> <br/>
+        <label for="weight"> Weight </label> <br/>
+        <input type="text" name="weight" id="weight" autocomplete="off" :placeholder="unicorn.weight" class="formulaire-unicorn__input"> <br/>
+        <label for="hobbies"> Hobbies </label> <br/>
+        <input type="text" name="hobbies" id="hobbies" autocomplete="off" :value="unicorn.hobbies" class="formulaire-unicorn__input"> <br/>
+        <label for="picture"> {{ txtPicture }} </label> <br/>
+        <form>
+            <input type="file" class="formulaire-unicorn__input-file formulaire-unicorn__input-file--padding" name="files" accept="image/png, image/jpeg">
+            <hr> 
+            <input type="submit" @click="actSubmit" class="formulaire-unicorn__input-button" :value="valeur">
+        </form>
+    </div>
+</template>
+<script>
+
+export default{
+    name: "FormulaireUnicorn",
+    props: {
+        unicorn: {
+            type: Object,
+            default: () => {return {name: "", birthyear: "", weight: "", hobbies: "", photo: null};}
+        },
+        valeur: {
+            type: String,
+            default: "Envoyer"
+        },
+        txtPicture: {
+            type: String,
+            default: "Picture"
+        }
+    },
+
+    methods: {
+        actSubmit(e) {
+            e.preventDefault();
+            this.$emit("act");
```

Comment:
tu dois véhiculer les data au composant parent, de cette façon : 
`this.$emit("act", this.unicorn)`